charts/vela-core/templates/defwithtemplate/apply-deployment.yaml

```diff
@@ -13,12 +13,8 @@ spec:
   schematic:
     cue:
       template: |
-        import (
-        	"strconv"
-        	"strings"
-        	"vela/kube"
-        	"vela/builtin"
-        )
+        import "vela/kube"
+        import "vela/builtin"
```

Comment:
Why are we missing the strconv and strings package import? Same comment across the board